roles/ara/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+# Keep 7 days worth of ARA data
+ara_prune_days: 7
+#Run offset from ansible-runner :07,:22,:37,:52
+ara_prune_cron_schedule: 7-59/15 
```

Comment:
This won't actually do what you expect, it will be parsed as (7-59)/15, which ends up being :15, :30, :45

From `man 5 crontab`:
```
Step  values  can  be  used in conjunction with ranges.  Following a range with ``/<number>'' specifies skips of
the number's value through the range.  For example, ``0-23/2'' can be used in the hours field to specify command
execution every other hour (the alternative in the V7 standard is ``0,2,4,6,8,10,12,14,16,18,20,22''). Steps are also
permitted after an asterisk, so if you want to say ``every two hours'', just use ``*/2''.
```

Let's just explicitly list the four times with: `7,22,37,52`